utils/ccdatofhir/ccda_to_fhir.bal

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// --------------------------------------------------------------------------------------------#
+// Source C-CDA to FHIR - Resource Mappings
+// --------------------------------------------------------------------------------------------#
+
+import ballerinax/health.fhir.r4;
+
+# Transform an C-CDA message to FHIR.
+#
+# + ccda - C-CDA message as a xml
+# + return - FHIR Bundle
+public function ccdaToFhir(xml<xml:Element|xml:Comment|xml:ProcessingInstruction|xml:Text> ccda) returns r4:Bundle|error {
+    r4:Bundle transformToFHIRResult = check transformToFhir(ccda);
+    return transformToFHIRResult;
+}
```

Comment:
```suggestion
public function ccdaToFhir(xml<xml:Element|xml:Comment|xml:ProcessingInstruction|xml:Text> ccda) returns r4:Bundle|error => transformToFhir(ccda);
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/bfbad6e342e0df400c6fffecc9682cf0f99e19b7